webclient/src/components/KnownHosts/KnownHosts.tsx

```diff
@@ -1,5 +1,4 @@
-// eslint-disable-next-line
-import React, { useCallback, useEffect, useState } from 'react';
```

Comment:
Doesnt `React` always have to be imported, even when unused? That's the reason for the disable lint line.